components/list/list-item-drag-drop-mixin.js

```diff
@@ -334,6 +335,28 @@ export const ListItemDragDropMixin = superclass => class extends superclass {
 				pointer-events: none;
 			}
 
+			:host([draggable]:not([selectable]):not([expandable])) .d2l-list-item-drag-area {
+				display: grid;
+				grid-template-columns: 
+					[start outside-control-start] minmax(0, min-content)
+					[color-start outside-control-end] minmax(0, min-content)
+					[expand-collapse-start color-end] minmax(0, min-content)
+					[control-start expand-collapse-end] minmax(0, min-content)
+					[control-end content-start] minmax(0, auto)
+					[content-end actions-start] minmax(0, min-content)
+					[end actions-end];
+			}
+			.d2l-list-item-drag-area [slot] {
+				grid-row: 1;
+			}
+			.d2l-list-item-drag-area [slot="content"] {
+				grid-column: content-start / actions-start;
+			}
+			.d2l-list-item-drag-area [slot="actions"] {
+				grid-column: actions-start / actions-end;
+				justify-self: end;
+			}
```

Comment:
In the other cases (e.g., selectable, etc), it was relatively easy to just put the content in the primary action container. This was not the case here, as the primary action container is in the `outside-control-action` grid slot and there were a lot of challenges around grid spacing.

I ended up re-creating the grid within this primary action, so that the content would have proper spacing and get the styles applied in list-item-mixin. This doesn't seem great and is a bit fragile, so let me know alternative ideas.